framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -3106,7 +3106,7 @@ public boolean validateTypeOf(Tree tree) {
             }
             break;
         default:
-            type = atypeFactory.getAnnotatedType(tree);
+        	type = atypeFactory.getDefaultedAnnotatedType(tree);
```

Comment:
>I'll try reproducing this error in a standard type system and will get back to you.
 Sounds good.  It might also be helpful if I knew the rules of the Universe type system.